libfreerdp/core/nla.c

```diff
@@ -935,21 +953,21 @@ BOOL nla_read_ts_credentials(rdpNla* nla, PSecBuffer ts_credentials)
 	}
 
 
-	/* TSCredentials (SEQUENCE) */
-	ber_read_sequence_tag(s, &length);
+		/* TSCredentials (SEQUENCE) */
+	ret = ber_read_sequence_tag(s, &length) &&
 
-	/* [0] credType (INTEGER) */
-	ber_read_contextual_tag(s, 0, &length, TRUE);
-	ber_read_integer(s, NULL);
+		/* [0] credType (INTEGER) */
+		ber_read_contextual_tag(s, 0, &length, TRUE) &&
+		ber_read_integer(s, NULL) &&
 
-	/* [1] credentials (OCTET STRING) */
-	ber_read_contextual_tag(s, 1, &length, TRUE);
-	ber_read_octet_string_tag(s, &ts_password_creds_length);
+		/* [1] credentials (OCTET STRING) */
+		ber_read_contextual_tag(s, 1, &length, TRUE) &&
+		ber_read_octet_string_tag(s, &ts_password_creds_length) &&
 
-	nla_read_ts_password_creds(nla, s);
+		nla_read_ts_password_creds(nla, s);
```

Comment:
Looks like you wanted to add return value checks for ber_read_contextual_tag, ber_read_integer, ber_read_octet_string_tag and nla_read_ts_password_creds but then forgot ?